apps/backend/tests/conftest.py

```
@@ -155,3 +166,98 @@
+        yield test_client
+    
+    # Cleanup
+    app.dependency_overrides.clear()
+
+
+@pytest.fixture
+async def async_client(test_session):
+    """Create async test client."""
+    def override_get_db():
+        return test_session
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)